packages/tokens/properties/color/fields.json

```diff
@@ -0,0 +1,18 @@
+{
+    "color": {
+        "fields": {
+            "error": {
+                "value": "{color.danger.500.value}"
```

Comment:
color.font.danger is not the same than color.danger.500

so the message color and input border color is not the same.

I keep color.font.danger ??

Reply:
Hi Corentin, we will discuss this with the design but yes I would say so because font colors have been tested for accessibility. Contrast should be good enough on white background. 

@nherwegh @caqueste

Reply:
fixed